internal/pkg/aws/cloudfront/cloudfront.go

```diff
@@ -5,4 +5,8 @@ package cloudfront
 const (
 	// CertRegion is the only AWS region accepted by CloudFront while attaching certificates to a distribution.
 	CertRegion = "us-east-1"
+
+	// S3BucketOriginDomainFormat is the Regex validation format for S3 bucket as CloudFront origin domain
+	// See https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/distribution-web-values-specify.html#DownloadDistValuesDomainName
+	S3BucketOriginDomainFormat = `.+\.s3.*\.\w+-\w+-\d+\.amazonaws\.com`
```

Comment:
That's a good question. s3-website I think is just so that you can have website endpoint vs rest API endpoint: https://docs.aws.amazon.com/AmazonS3/latest/userguide/WebsiteEndpoints.html#WebsiteRestEndpointDiff